CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+## 1.0rc1
+
+### New Additions
+
+* Bamboo allows adapters to support attachments! (https://github.com/thoughtbot/bamboo/pull/156)
+* Add `MailgunAdapter` attachment support (https://github.com/thoughtbot/bamboo/commit/d47833194833e6a1cb9f9cb715be0742e55f5fd8)
+* Add support for `replyto` header in `SendGridAdapter` (https://github.com/thoughtbot/bamboo/pull/254)
+* Raise if email has attachments and adapter doesn't support them (https://github.com/thoughtbot/bamboo/commit/ce2249c9854a79148ecf91f877ae26142c83184b)
```

Comment:
I used to take more time to make these look nice, but it took too much time for a very small benefit. I think direct links like this are fine. Thoughts?